qubes/tests/app.py

```diff
@@ -315,6 +317,20 @@ def test_100_get_new_unused_qid(self):
 
         self.vms.get_new_unused_qid()
 
+    def test_999999_get_new_unused_dispid(self):
+        with mock.patch('random.SystemRandom') as random:
+            random.return_value.randrange.side_effect = [11, 22, 33, 44, 55, 66]
+            # Testing overal functionality
+            self.assertEqual(self.vms.get_new_unused_dispid(), 11)
+            self.assertEqual(self.vms.get_new_unused_dispid(), 22)
+            self.assertEqual(self.vms.get_new_unused_dispid(), 33)
+            # Testing no reuse
+            self.vms._recent_dispids[44] = 0
```

Comment:
This works only because CI runs tests in a fresh VM, so `time.monitonic()` is quite small. But it will fail if you run it on a system with over a week of uptime... Use `time.monitonic()` here? Or patch it with a constant value?

Reply:
Done

Reply:
Let me also add time.monotonic to the other test (66)